modules/apps/foundation/frontend-js/frontend-js-loader-modules-extender/src/main/java/com/liferay/frontend/js/loader/modules/extender/internal/npm/flat/FlatNPMBundleProcessor.java

```diff
@@ -108,6 +127,13 @@ private String _getResourceContent(
 		return Arrays.asList(dependencies);
 	}
 
+	/**
+	 * Process a dependencies type entry of a package.json file adding them to
```

Comment:
@izaera Is the `dependencies type entry of a package.json file` an actual property found in the package.json? For properties, we typically try and emphasize their name so they're easy to distinguish. I'll surround it with `<code>` tags if it's an actual property.